test/rostest_px4_run.sh

```diff
@@ -3,7 +3,7 @@
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 PX4_SRC_DIR=${DIR}/..
 
-source /opt/ros/kinetic/setup.bash
+source /opt/ros/${ROS_DISTRO}/setup.bash
```

Comment:
Default to kinetic if `ROS_DISTRO` doesn't exist?

Reply:
That's definitely a good idea.

Reply:
I didn't consider use outside of CI. Fixed.

Reply:
I don't think this is a good idea. What is the rationale for setting a default distro if we don't know if any is setup on the system. Also, if using ROS tools, the env variables have to always be set, which includes the ROS_DISTRO.

Reply:
This is a chicken and egg problem. `ROS_DISTRO` isn't set until we source `setup.bash`.

I see no problem setting a default version to correspond with the officially supported version that we use for testing. This is a testing script. Not a tools script.